Projects/Application/Sources/Common/Network/Http/Configuration.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Created by Maxim Berezhnoy on 2019-02-12.
+// Licensed under the MIT license
+//
+//
+
+import Foundation
+
+extension Http {
```

Comment:
Maybe it is better to extract HTTP client to a separate dependency so HTTP namespace can be used. Also, HTTP needs to be all capital.

Reply:
> Maybe it is better to extract HTTP client to a separate dependency so HTTP namespace can be used.

Great idea! I was, indeed, simulating namespace by using extension; but that approach is flawed as Swift, for instance, does not support type declaration inside the extensions.
I will move it to the separate component.

> Also, HTTP needs to be all capital
That was a conscious decision. I am aware that HTTP is an abbreviation, but style-wise I prefer the `Http` name.

Reply:
I still think that `Http` looks a bit ugly. I see no problem in all capital version. Especially if it is going to be extracted in a separate framework, so it can be called HTTP.

Reply:
It's a matter of preference, really. I don't like excessive capitalization. But since all capital version is technically correct, I will use that name instead.

Reply:
Fixed